Formula/hledger.rb

```diff
@@ -5,44 +5,32 @@ class Hledger < Formula
 
   desc "Command-line accounting tool"
   homepage "https://hledger.org/"
-  url "https://hackage.haskell.org/package/hledger-1.14.2/hledger-1.14.2.tar.gz"
-  sha256 "849a6e0683192ec504da9a631ddfc82e04973583f4a028fd39b8cdac2efe29ea"
+  url "https://hackage.haskell.org/package/hledger-1.15.1/hledger-1.15.1.tar.gz"
 
```

Comment:
please add the correct `sha256` for this file. Try: `curl -L https://hackage.haskell.org/package/hledger-1.14.2/hledger-1.14.2.tar.gz | shasum -a 256`